appveyor.yml

```diff
@@ -13,6 +13,14 @@ environment:
     - PYTHON: "C:\\Python35-x64"
     - PYTHON: "C:\\Python36"
 
+skip_commits:
+  files:
+    - .github/
+    - .travis/
```

Comment:
`.travis.yml` should probably be ignored too.

Reply:
Indeed.